tailormap-components/projects/bridge/src/tailor-map.service.ts

```diff
@@ -43,7 +47,7 @@ export class TailorMapService {
     const mc = vc.mapComponent;
     const map = mc.getMap();
     map.addListener('ON_LAYER_VISIBILITY_CHANGED', (object, event) => {
-      this.layerVisibilityChanged$.next(event);
```

Comment:
These events indeed did not trigger change detection right away. Running inside the ngZone here removes the need to use ngZone where the Observable is subscribed to